docs/concepts/registries.md

```diff
@@ -112,7 +109,7 @@ The rules for `@ObjectHolder` are as follows:
 * If no other errors or exceptions occur, the field will be injected
 * If all of the above rules do not apply, no action will be taken (and a message may be logged)
 
-`@ObjectHolder`-annotated fields are injected with their values after their corresponding registry's `RegistryEvent$Register` event is fired, along with the `RegistryObject`s.
+`@ObjectHolder`-annotated fields are injected with their values after `RegisterEvent` is fired for their registry, along with the `RegistryObject`s.
```

Comment:
This section on `@ObjectHolder` needs to be revisited, since there has been changes to the annotation due to the removal of the Forge registries' reliance on type information.